Pod/Classes/Terminals/Utility/GlobalMobileSDK/GMSRequestHelper.swift

```diff
@@ -91,13 +91,13 @@ class GMSRequestHelper {
                                           posReferenceNumber: posReferenceNumber,
                                           operatingUserId: operatingUserId)
     }
-    
+
     public static func buildCreditReturnRequest(builder: GMSCreditReturnBuilder) -> Transaction? {
         let total: Decimal? = builder.amount as Decimal?
         let posReferenceNumber: String? = builder.referenceNumber
         let transactionId: String? = builder.transactionId
 
-        return ReturnTransaction.returnWithReference(total: decimalToUint(total),
+        return ReturnTransaction.returnWithReference(clientTransactionId: "", total: decimalToUint(total),
```

Comment:
The value for `clientTransactionId` should be obtained from the builder when present.